functions/brinks/src/constants.js

```diff
@@ -0,0 +1,25 @@
+const constants = () => ({
+  that: {
+    productType: {
+      ticket: 'TICKET',
+      membership: 'MEMBERSHIP',
+      partnership: 'PARTNERSHIP',
+      food: 'FOOD',
+    },
+  },
+  stripe: {
+    checkoutMode: {
+      payment: 'payment',
+      subscription: 'subscription',
+    },
+    subscriptionStatus: {
+      active: 'active',
+      cancelled: 'canceled',
+      pastDue: 'past_due',
+      unpaid: 'unpaid',
+      incomplete: 'incomplete',
+      incompleteExpired: 'incomplete_expried',
+    },
+  },
+});
+export default constants();
```

Comment:
does it really need to be a function? just return the object.

Reply:
I was following the envConfig.js

Reply:
the difference being, your constants don't call any functions, they're just strings, so nothing to fail. the env config will fail on creation as it calls functions to validate it's settings.

Reply:
fixed in commit 7d382d0